avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -28,3 +26,14 @@ aimecDir = Outputs/com1DFA/
 calcPressureLimit = 10
 # width of the avalanche path domain for computations in Aimec
 domainWidth = 600
+
+
+[PARAMATERVAR]
+# True if parameter variation of MU option shall be included
+flagVarPar = False
+# parameter to be varied, current option: Mu (friction parameter)
+varPar = Mu
```

Comment:
can we change the Mu to something like depth? will it work?

Reply:
I added the option to change the release area thickness